lib/auth0/api/v2/users.rb

```diff
@@ -220,19 +221,19 @@ def remove_user_roles(user_id, roles)
           raise Auth0::MissingUserId, 'Must supply a valid user_id' if user_id.to_s.empty?
           validate_strings_array roles
           path = "#{users_path}/#{user_id}/roles"
-          delete(path, { roles: roles })
+          delete_with_body path, roles: roles
```

Comment:
I see inconsistencies on the syntax when calling a function with parameters.

`delete(path, {param: value}` vs `delete path, param:value`. 

Should we add a linter as a precommit for this repo as well or is this change on purpose?

Reply:
There is a linter that I run, `rubocop`, but it doesn't pick this up